python/django_bolt/views.py

```
@@ -41,7 +41,7 @@ class APIView:
         status_code: Default status code for responses (can be overridden per-method)
     """
 
-    http_method_names = ["get", "post", "put", "patch", "delete", "head", "options"]
+    http_method_names = ["get", "post", "put", "patch", "delete", "head", "options", "query"]
```

Ruff (0.15.20)

[warning] Mutable default value for class attribute

(RUF012)